neonvm/config/common/controller/manager.yaml

```diff
@@ -81,10 +81,10 @@ spec:
         # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
         resources:
           limits:
-            cpu: 500m
-            memory: 128Mi
+            cpu: 2000m
+            memory: 1024Mi
           requests:
-            cpu: 10m
-            memory: 64Mi
+            cpu: 500m
+            memory: 512Mi
```

Comment:
I've tuned it several times, final size was decided after load test with 1000 VMs migrating simultaneously. Controller was killed by OOMKiller several times. I see no reason create separate PR for such small changes.

Reply:
The reason to create a separate PR for small changes like this is so that:

1. It's easier for me to review
2. the PR isn't "all or nothing" — currently, I feel bad asking for changes to this PR because it's delaying _everything_. But ultimately that's not on me; the code standards don't change depending on the size of the PR, so bundling everything together means that everything waits longer.

Reply:
added to follow ups list

Reply:
If you are leaving this change as a follow-up, this PR has not yet been amended to leave the values as-is. Is that intentional?

Reply:
just pushed commit with revert